run_app.py

```diff
@@ -0,0 +1,182 @@
+# Copyright 2016-2021 The Van Valen Lab at the California Institute of
+# Technology (Caltech), with support from the Paul Allen Family Foundation,
+# Google, & National Institutes of Health (NIH) under Grant U24CA224309-01.
+# All rights reserved.
+#
+# Licensed under a modified Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.github.com/vanvalenlab/deepcell-applications/LICENSE
+#
+# The Work provided may be used for non-commercial academic purposes only.
+# For any other use of the Work, including commercial use, please contact:
+# vanvalenlab@gmail.com
+#
+# Neither the name of Caltech nor the names of its contributors may be used
+# to endorse or promote products derived from this software without specific
+# prior written permission.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""An example script for Multiplex Segmentation using deepcell.applications."""
+
+import argparse
+import logging
+import os
+import sys
+import timeit
+
+import numpy as np
+import tifffile
+
+import deepcell_applications as dca
+
+
+def get_arg_parser():
```

Comment:
Looks like there's a formatting issue with the help text that comes up after an invalid argument is specified.
![image](https://user-images.githubusercontent.com/13770365/129110861-3d0df6b6-0882-4138-8f77-2c4bf1b90326.png). 

The --nuclear-image arg should be on a new line, and is missing brackets

Reply:
I'll look into the spacing but there are no brackets because it is the only one that is required. Everything else has valid default values.

Reply:
This has stumped me. It seems that `nuclear-image` is on its own line because its a "required" option (which apparently is not an advised pattern, as anything with `--` prefixed is considered "optional" and not "required").

The order is also affected because of the parent parser we use to hold common options.

Reply:
Yeah, doesn't seem like there's a better option